### PR TITLE
WIP: Added dynamic resizing to support orientation changes

### PR DESCRIPTION
**Pull request by knly**
I added Auto Layout constraints to added pages that are dynamically updated when the `BWWalkthroughViewController`'s view is resized, particularly when the orientation changes.

### CODE REVIEW

**Comment by kevinonik** [comment]
Whats the update on this? Need landscape orientation

**Comment by ariok** [comment]
I check it in a couple of  days. Sorry guys I was very busy with Linia :P (linia.blackrobotgames.com)

**Comment by ariok** [comment]
@knly just checked this code. It seems navigation buttons are not longer working. If you try to click next/previous buttons the navigation to the next/previous view controllers doesn't start anymore. The problem seems to be in the `goToPage` method, scroll view is no longer animating to the right position, I cannot understand why though...

File: Pod/Classes/BWWalkthroughViewController.swift [hunk 1]
```diff
@@ -206,13 +212,17 @@ At the moment it's only used to perform custom animations on didScroll.
...
+        self.widthConstraints.append(widthConstraint)
+        self.heightConstraints.append(heightConstraint)
         
         // - Generic cnst
         
-        vc.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:[view(h)]", options:[], metrics: metricDict, views: ["view":vc.view]))
-        vc.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:[view(w)]", options:[], metrics: metricDict, views: ["view":vc.view]))
-        scrollview.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|-0-[view]|", options:[], metrics: nil, views: ["view":vc.view,]))
+        vc.view.addConstraints([ widthConstraint, heightConstraint ])
+        scrollview.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|-0-[view]", options:[], metrics: nil, views: ["view":vc.view,]))
```
Comment (gnatok) [inline Pod/Classes/BWWalkthroughViewController.swift]: Mistake here @ariok @knly. Instead of `"V:|-0-[view]"` has to be `"V:|-0-[view]|"`

**Comment by gnatok** [comment]
kills @ariok @knly simple error kills next/prev button. Here https://github.com/ariok/BWWalkthrough/pull/60/files#diff-f82ac6ba0676c48e8d8d7ae32483e8f1R225

**Comment by ariok** [comment]
It has been implemented with #80.

**Pull request closed without merging**